wordpress-mobile/WordPress-iOS
# Merge 15.2 code freeze into develop

Merges 15.2 code freeze changes into `develop`.

**To Test**
- Verify that all the changed version numbers go up
- Verify that all the changed beta pods no longer say beta
